src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -583,7 +583,7 @@ def construct_containerapp_for_post_process(self, containerapp_def, r):
             registries_def["server"] = self.get_argument_registry_server()
             registries_def["identity"] = self.get_argument_registry_identity()
             safe_set(containerapp_def, "properties", "configuration", "registries", value=[registries_def])
-        return containerapp_def
+            return containerapp_def
```

Comment:
with this change, there is no return out of if

Reply:
Don't think it would not matter functionality wise since we are always returning `containerapp_def` on line 586.(This is how it currently is in [main](https://github.com/Greedygre/azure-cli-extensions/blob/xinyu/containerapp_create_refactor/src/containerapp/azext_containerapp/containerapp_decorator.py#L556) I agree stylistically, it is better to return out of the if statement instead.